cmake/Platform/Mac/runtime_dependencies_mac.cmake.in

```diff
@@ -235,7 +186,7 @@ else() # Non-bundle case
 
     if(depends_on_python)
         # RPATH fix python
-        execute_process(COMMAND "${LY_INSTALL_NAME_TOOL}" -change @rpath/Python @rpath/Python.framework/Versions/Current/Python "@target_file@")
+        execute_process(COMMAND "${LY_INSTALL_NAME_TOOL}" -change @rpath/Python @rpath/.. "@target_file@")
```

Comment:
Shouldn't this point to a lib instead of the directory `..`? The `-change` option `Changes the dependent shared library install name old to new in the specified Mach-O binary.`

Reply:
Yeah, I only just realized 'Python' was the lib name, not the Path